src/Evaluator/NodeEvaluator/MethodNodeEvaluator/MethodNodeHandlers/ExcellentHandler.ts

```diff
@@ -32,10 +32,11 @@ export class ExcellentHandler implements MethodNodeHandler {
   }
 
   public remove_fist_word(string: string): string {
```

Comment:
This may or may not be a string, such as in the case where a date is passed. So the type should be unknown. This is likely too big of a task for this PR, but it should be considered for all Handlers, as the user can pass wrong types all over the code.